cloudferrylib/base/exception.py

```diff
@@ -21,6 +21,10 @@ class ImageDownloadError(Exception):
     pass
 
 
+class ErrorCheckCloud(Exception):
```

Comment:
Either inherit it from `AbortMigrationError` or use `AbortMigrationError` directly. Not sure why new exception type is needed.